sources/packages/api/src/auth/jwt.strategy.ts

```diff
@@ -41,11 +41,11 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
     // also as needed to be validate using a decorator/annotation.
     userToken.roles = [];
     if (payload.resource_access) {
-      Object.keys(payload.resource_access).forEach((value) => {
-        payload.resource_access[value].roles.forEach((roleValue: string) => {
+      payload.resource_access[payload.azp].roles.forEach(
```

Comment:
Do we need this processing to happen? I believe that the token is already in the format that we expected, right?

Reply:
yup.i will update